cdc/log.cc

```diff
@@ -1833,6 +1846,7 @@ class transformer final : public change_processor {
                     _static_row_state[&c] = std::move(*maybe_cell_view);
                 }
             }
+            _is_update = true;
```

Comment:
Nit: probably could have been better to call it e.g. `_updates_existing` or something like that, to clarify that we are updating something existing and we are not meaning a CQL UPDATE here.